src/elements/DestructiveDialog/DestructiveDialog.less

```diff
@@ -0,0 +1,32 @@
+/**
+ * Destructive Dialog styles
+ */
+
+ @ui-destructive-dialog-font: @ui-font-sans-serif;
+ @ui-destructive-dialog-letter-spacing: 0.02em;
+
+.ui-destructive-dialog-container {
+	font-family:  @ui-destructive-dialog-font;
+	letter-spacing:  @ui-destructive-dialog-letter-spacing;
+	height: 186px;
```

Comment:
Let's remove height/width from the stylesheet and make them props in the component that default to these values, and just attach it directly to the `style` attribute on the element.